.github/workflows/build.yml

```
@@ -77,6 +77,15 @@ jobs:
             ${{ env.TARGET_NAME }}${{ matrix.ext }}
             wallet.svg
             manifest.json
+      - name: Check the manifest version
+        if: ${{ inputs.tag_name != '' }}
+        run: |
+          sudo apt-get install -y jq
+          version=$(jq -r '.key' manifest.json)
```

Isn't it supposed to be `version` ? 🤔 

```suggestion
          version=$(jq -r '.version' manifest.json)
```

---

Based on this:
https://stackoverflow.com/questions/61919141/read-json-file-in-github-actions#comment126579492_61919791